src/sections/Home/Partners-home/index.js

```diff
@@ -44,7 +44,7 @@ const Projects = () => {
         {partners.map((partner, index) => (
           <Link className="partner-card" to={partner.imageRoute} key={index}>
             <div className={partner.innerDivStyle}>
-              <img className="partner-image" loading="lazy" src={partner.imageLink} alt={partner.name} width={partner.imageWidth} height={partner.imageHeight} />
+              <img className="partner-image" id={partner.id} loading="lazy" src={partner.imageLink} alt={partner.name} width={partner.imageWidth} height={partner.imageHeight} />
```

Comment:
@Srinja333 thanks. 

Fewer lines of code is almost always better. Will you restructure this to simply use the existing `partner.name` attribute instead of creating a new one?

Reply:
Can I try?